iOSClient/Images.xcassets/audioPlay.imageset/Contents.json

```diff
@@ -1,15 +1,15 @@
 {
   "images" : [
     {
-      "idiom" : "universal",
-      "filename" : "play.pdf"
+      "filename" : "play.pdf",
```

Comment:
This Icon change can also be moved to Icons PR as mentioned in dashboard theming PR.